test/functional/host_collection/host_test.rb

```diff
@@ -0,0 +1,40 @@
+require_relative '../test_helper'
```

Comment:
I think this file should technically be `hosts_test.rb` since the subcommand is `hosts`.